zhaquirks/sinope/__init__.py

```diff
@@ -52,7 +52,7 @@ class ButtonAction(t.enum8):
             ATTRIBUTE_ID: 84,
             ATTRIBUTE_NAME: ATTRIBUTE_ACTION,
             BUTTON: TURN_ON,
-            VALUE: ButtonAction.Pressed_on,
+            VALUE: int(ButtonAction.Pressed_on),
```

Comment:
Good catch, sorry about it.

I personally prefer `VALUE: ButtonAction.Pressed_on.value` to match the syntax when we emit.

Another fix would be to change to `DESCRIPTION: ButtonAction.Pressed_on.name`, but I don't have a strong preference.

Reply:
So do I :)

Problem is that when I tested using `.value`, it turned out that it's of type `<class 'zigpy.types.basic.uint8_t'>` and not an `int`! This still made Voluptuous raise an invalid type exception, so I opted for the next simplest solution, coercing to int.